### DISCUSSION TITLE
Log Out / Update Credentials

### DISCUSSION THREAD

**Discussion by boasist** [greghesp/ha-bambulab#772] [Q&A]
Since Bambu has an issue with apple relay accounts, I've changed everything and moved accounts entirely. Yet since, the integration isn't working...of course.

I've tried disabling, reloading, and force updating, but nothing prompts for a password anymore. 

How can I log out/update my credentials?

**Comment by AdrianGarside** [comment]
The latest version now allows you to choose whether to re-use existing working credentials or enter fresh ones. Please try it out.
